### DISCUSSION TITLE
hyperparameters

### DISCUSSION THREAD

**Discussion by VAIBHAVSHAKYA** [allenai/allennlp#5687] [Q&A]
what are the hyperparameters for coref ?

**Comment by dirkgr** [comment]
The complete training config is here: https://raw.githubusercontent.com/allenai/allennlp-models/main/training_config/coref/coref_spanbert_large.jsonnet. I found that on the model card for the coref model in the demo: https://demo.allennlp.org/coreference-resolution.
